flow/environment.py

```diff
@@ -225,7 +225,12 @@ def add_args(cls, parser):
             The argument parser where arguments will be added.
 
         """
-        return
+        parser.add_argument(
```

Comment:
I moved all the "walltime" parser options to the base class `ComputeEnvironment`. We will remove this in 0.14, and this unifies the implementation in the meantime so that deleting it in 0.14 is easier. Until then, every environment has a `--walltime` option, and that also allows us to test the output behavior with the `StandardEnvironment`/`slurm.sh` in the walltime project tests.